### PR TITLE
Fix meta

### PR DESCRIPTION
**Pull request by sisuresh**
# Description

# Checklist
- [ ] Reviewed the [contributing](https://github.com/stellar/stellar-core/blob/master/CONTRIBUTING.md#submitting-changes) document
- [ ] Rebased on top of master (no merge commits)
- [ ] Ran `clang-format` v8.0.0 (via `make format` or the Visual Studio extension)
- [ ] Compiles
- [ ] Ran all tests
- [ ] If change impacts performance, include supporting evidence per the [performance document](https://github.com/stellar/stellar-core/blob/master/performance-eval/performance-eval.md)

### CODE REVIEW

File: src/ledger/LedgerTxn.h
Comment (sisuresh) [inline src/ledger/LedgerTxn.h:578]: I made this an optional parameter in the interest of time. We're going to remove this anyways so it should be fine.

File: src/ledger/LedgerTxn.cpp
Comment (SirTyson) [inline src/ledger/LedgerTxn.cpp:1461]: Need to add an expiration check above in `LedgerTxn::Impl::getNewestVersion` now that `mEntry` may contain expired entries since `load` may make an expired entry active. This also applies in `getNewestVersionEntryMap`.

**Comment by sisuresh** [comment]
Marked as draft so we don't merge this in prior to when the 19.13.0 release is cut.

File: src/ledger/LedgerTxn.cpp
Comment (sisuresh) [reply src/ledger/LedgerTxn.cpp:1485]: Good point. Fixed

**Review by SirTyson — APPROVED** [review]
LGTM after the `src/transactions/RestoreFootprintOpFrame.cpp` issue with last modified ledgerSeq is fixed. We're not gonna keep this so you can just `loadWithoutRecord`, then if you actually update something just `load` another copy.

**Review by SirTyson — APPROVED** [review]
LGTM

**Comment by sisuresh** [comment]
Resolved by https://github.com/stellar/stellar-core/pull/3857

**Pull request closed without merging**
